src/Octoshift/Handlers/DownloadLogsCommandHandler.cs

```diff
@@ -63,8 +63,6 @@ public async Task Handle(DownloadLogsCommandArgs args)
             _log.LogInformation($"MIGRATION LOG FILE: {args.MigrationLogFile}");
         }
 
-        args.MigrationLogFile ??= $"migration-log-{args.GithubOrg}-{args.GithubRepo}.log";
-
         if (FileExists(args.MigrationLogFile))
```

Comment:
We need to move this down as well, since we figure out the filename in a different place.